lib/split/dashboard/public/style.css

```diff
@@ -258,7 +258,7 @@ body {
   color: #408C48;
 }
 
-a.button, button, input[type="submit"] {
+.experiment a.button, .experiment button, .experiment input[type="submit"] {
```

Comment:
Already in our code: https://github.com/clio/split/blob/master/lib/split/dashboard/public/style.css#L261